ocp_utilities/operators.py

```diff
@@ -169,6 +169,10 @@ def install_operator(
             raise ValueError("source must be provided if not using iib_index_image")
 
     operator_namespace = operator_namespace or name
+
+    if name == "nfd":
```

Comment:
You need to pass `target_namespaces` to the function if you want target_namespaces

Reply:
ok, I'll modify `ocp-addons-operators-cli` then.
thanks